data/example_gen_snippets.json

```diff
@@ -0,0 +1,168 @@
+{
```

Comment:
Maybe rename to `documents_snippets`? This name make me thinks that this are example of generated snippets.